dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -4,7 +4,7 @@
             "tunnel_type":"IPINIP",
             "src_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
             "dst_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
-{% if minigraph_hwsku in [ 'ACS-MSN2700', 'ACS-MSN2410', 'ACS-MSN2100' ] %}
+{% if asic_type == 'mellanox' %}
```

Comment:
wondering if tested on any other asic types, or if we want to specify broadcom/mellanox.